CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# Phootwork contributing guide
+
+If you're reading this document, it is likely that you have decided to contribute to this project.
+Therefore, first of all, we want to warmly thank you! The Open Source Software grows and improves
+thanks to people like you!
+
+Contributing to a project is a process with its own rules and we try to explain them along this document.
+
+## Quick start for experienced programmers
+
+1. Fork, clone and apply your patches. See the [directory structure](#understanding-the directory-structure) explanation
+if needed and don't forget to write tests.
+2. Run the test suite ` vendor\bin\phpunit ` and fix all red tests.
+3. Run [psalm](https://psalm.dev/) `vendor\bin\psalm` and fix all errors and issues.
+4. Fix the coding standard `composer cs-fix`.
+5. [Squash](https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/) some commits if necessary
+and submit your pull request.
+
+## Requirements
+
+To contribute to this project, you should have an good knowledge of PHP (of course) and a basic knowledge of
+[git](https://git-scm.com/), [Github](https://github.com/) and the dependency manager [composer](https://getcomposer.org/). 
+
+## First step: clone the project and install the dependencies
+
+The Phootwork project is a collection of single libraries that can be installed separately. Anyway, all the development
+process takes place in the global repository https://github.com/phootwork/phootwork .
+
+So, the first step is to fork the [Phootwork repository](https://github.com/phootwork/phootwork) on Github and clone it
```

Comment:
`phootwork` is case-sensitive and lower-case (sorry that's me 🙈)

Reply:
Fixed